internal/debugagent/debugagent.go

```diff
@@ -0,0 +1,370 @@
+package debugagent
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"time"
+
+	"github.com/agentuity/cli/internal/tools"
+	"github.com/agentuity/go-common/logger"
+	"github.com/anthropics/anthropic-sdk-go"
+	"github.com/charmbracelet/glamour"
+)
+
+// NOTE: I think we should be able to use that fancy go:embed thing here
+// but the import gets nuked when we build the CLI, so doing this nasty
+// init() thing instead.
+var systemPrompt string
+
+func init() {
+	_, file, _, _ := runtime.Caller(0)
+	base := filepath.Dir(file)
+	p := filepath.Join(base, "./debug-system-prompt.txt")
+	if data, err := os.ReadFile(p); err == nil {
+		systemPrompt = string(data)
+	}
+}
```

Comment:
you need to use a blind import:

```suggestion
	_ "embed"
)

//go:embed debug-system-prompt.txt
var systemPrompt string
```

if your editor removes that still thats an issue with your config.